.github/workflows/pr.yaml

```diff
@@ -1,13 +1,15 @@
 # Copyright (c) 2023, NVIDIA CORPORATION.
-name: pr
+name: Build and test
 
 on:
   push:
     branches:
       - "pull-request/[0-9]+"
+    tags:
+      - 'v*'
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
   cancel-in-progress: true
```

Comment:
In theory this would mean if two tags are pushed concurrently, only the later running job would end up uploading a wheel. Probably this is unlikely to happen because we don't push tags very often (I think?)

I am also not sure that the `event_name` part disambiguates anything, since in both a branch-based push or a tag-based push the `event_name` is just `"push"`. Am I missing something?

Reply:
@wence- Do you have a fix for these issues in mind?

Reply:
The second issue might not be a huge concern yet either if the only drawback is that release jobs could compete for resources with PR jobs IMO.